x-pack/plugins/maps/server/mvt/util.ts

```diff
@@ -0,0 +1,10 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export function isAbortError(error: Error) {
+  return error.name === 'RequestAbortedError';
```

Comment:
I'd use `error instanceof errors.RequestAbortedError` to have a proper type check for the error